### ISSUE TITLE
Set default resolutions for Wifi/mobile

### ISSUE THREAD

**Issue by GUImodder** [Frontesque/VueTube#44]
Hi guys,

Vuetube seems like a great project and imho is the future of youtube on android.

Please consider that data caps are a huge issue in many parts of the world. Youtube's stock app was absolutely awful when it came to selecting default resolutions for wifi and mobile (mostly using unnecessary amounts of data on mobile while offering low resolutions on wife where data is less of an issue).

Vanced had this feature down: simply enable a set resolution for wife vs. mobile.

Please implement this in exactly the same simple and transparent way Vanced did.

BTW: is there a way to support this project with donations?

**Comment by Frontesque** [comment]
I hadn't even thought of this, but it sounds really useful. I assume that you mean that it selects a default resolution based on if the device is on WiFi and a different resolution if it is on Mobile Data? 

If so, this would actually be useful for myself as I have very limited data 😂

**Comment by GUImodder** [comment]
> I assume that you mean that it selects a default resolution based on if the device is on WiFi and a different resolution if it is on Mobile Data

Exactly! This is how this it looked in a recently discontinued youtube client mod ;-)

![Screenshot_20220315-174156](https://user-images.githubusercontent.com/15251611/158428833-22dcf8c5-1cb1-45cf-a4f2-27d40c33ef5d.jpeg)

Step 1


![Screenshot_20220315-174201](https://user-images.githubusercontent.com/15251611/158428855-6d8be354-6991-4c3f-b6a2-a6569c687aac.jpeg)

Step 2

Pretty much worked like a charm and saved me 100s of GBs using youtube while working out of listening to podcasts in my car.

**Comment by duckyondiscord** [comment]
This would be an amazing feature.
